docs/user-guide/odin_simulation.ipynb

```diff
@@ -0,0 +1,1049 @@
+    "\n",
+    "normalized_sim_wavelength_sample = sim_wavelength_binned_sample.bins.sum() / (\n",
+    "    sim_wavelength_binned_ob.bins.sum() + sc.scalar(1, unit='counts')\n",
+    ")\n",
+    "normalized_sim_wavelength_sample.coords['wavelength'] = (\n",
+    "    normalized_sim_wavelength_sample.coords['sim_wavelength']\n",
+    ")\n",
+    "normalized_sim_wavelength_sample = normalized_sim_wavelength_sample.rename_dims(\n",
+    "    {'sim_wavelength': 'wavelength'}\n",
+    ")\n",
+    "\n",
+    "pp.plot(\n",
+    "    {\n",
+    "        'True Wavelength': normalized_sim_wavelength_sample.sum('pixel_id'),\n",
+    "        'Naiively Calculated Wavelength': normalized_sample.sum('pixel_id'),\n",
+    "    },\n",
+    "    title=\"Normalized Sample Image\",\n",
+    "    grid=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11",
+   "metadata": {},
+   "source": [
+    "## Region of Interest\n",
+    "\n",
+    "### Merged ROIs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import plopp as pp\n",
+    "from plopp.widgets.tools import ToggleTool\n",
+    "from mpltoolbox import Rectangles\n",
+    "\n",
+    "\n",
+    "class RectInfo(sc.DataGroup):\n",
+    "    @property\n",
+    "    def min_x(self) -> sc.Variable:\n",
+    "        return self['min_x']\n",
+    "\n",
+    "    @property\n",
+    "    def max_x(self) -> sc.Variable:\n",
+    "        return self['max_x']\n",
+    "\n",
+    "    @property\n",
+    "    def min_y(self) -> sc.Variable:\n",
+    "        return self['min_y']\n",
+    "\n",
+    "    @property\n",
+    "    def max_y(self) -> sc.Variable:\n",
+    "        return self['max_y']\n",
+    "\n",
+    "    def __init__(\n",
+    "        self,\n",
+    "        min_x: sc.Variable,\n",
+    "        max_x: sc.Variable,\n",
+    "        min_y: sc.Variable,\n",
+    "        max_y: sc.Variable,\n",
+    "    ):\n",
+    "        super().__init__(\n",
+    "            {'min_x': min_x, 'max_x': max_x, 'min_y': min_y, 'max_y': max_y}\n",
+    "        )\n",
+    "\n",
+    "\n",
+    "def _get_rect_info(artist, figure) -> RectInfo:\n",
+    "    \"\"\"\n",
+    "    Convert the raw rectangle info to a ``RectInfo`` object.\n",
+    "    each axis, and values with units.\n",
+    "    \"\"\"\n",
+    "    x_range = sc.array(\n",
+    "        dims=['x'],\n",
+    "        values=[artist.xy[1], artist.xy[1] + artist.height],\n",
+    "        unit=figure.canvas.units['x'],\n",
+    "    )\n",
+    "    y_range = sc.array(\n",
+    "        dims=['y'],\n",
+    "        values=[artist.xy[0], artist.xy[0] + artist.width],\n",
+    "        unit=figure.canvas.units['y'],\n",
+    "    )\n",
+    "    return RectInfo(x_range.min(), x_range.max(), y_range.min(), y_range.max())\n",
+    "\n",
+    "\n",
+    "class MergedRectanglesTool(ToggleTool):\n",
```

Comment:
I also think this code should go in a submodule, so we can just simply import the tool in the notebook?

Reply:
Yeah. Then I'll do that in plopp maybe...?